rke2/ayekan/exporters/snmp-generator/Makefile

```diff
@@ -18,43 +18,28 @@ MIB_PATH := 'mibs'
 
 CURL_OPTS ?= -L -sS --retry 3 --retry-delay 3 --fail -A "Prometheus SNMP generator Makefile"
 
-APC_URL             := https://download.schneider-electric.com/files?p_enDocType=Firmware&p_File_Name=powernet451.mib&p_Doc_Ref=APC_POWERNETMIB_451_EN
+APC_URL             := https://download.schneider-electric.com/files?p_Doc_Ref=APC_POWERNETMIB_EN&p_enDocType=Firmware&p_File_Name=powernet457.mib
 ARISTA_URL          := https://www.arista.com/assets/data/docs/MIBS
-CISCO_URL           := https://raw.githubusercontent.com/cisco/cisco-mibs/2d465cce2de4e67a3561d8e41e4c99b597558d4b/v2
+CISCO_URL           := https://raw.githubusercontent.com/cisco/cisco-mibs/05dbf50226f7df5f52fd2dd1a9c17759273fa0d0/v2/
+CISCO_URL2           := https://raw.githubusercontent.com/cisco/cisco-mibs/05dbf50226f7df5f52fd2dd1a9c17759273fa0d0/v1/
 DELL_URL            := https://dl.dell.com/FOLDER11196144M/1/Dell-OM-MIBS-11010_A00.zip
 IANA_CHARSET_URL    := https://www.iana.org/assignments/ianacharset-mib/ianacharset-mib
 IANA_IFTYPE_URL     := https://www.iana.org/assignments/ianaiftype-mib/ianaiftype-mib
 IANA_PRINTER_URL    := https://www.iana.org/assignments/ianaprinter-mib/ianaprinter-mib
-KEEPALIVED_URL      := https://raw.githubusercontent.com/acassen/keepalived/v2.2.8/doc/KEEPALIVED-MIB.txt
 VRRP_URL            := https://raw.githubusercontent.com/acassen/keepalived/v2.2.8/doc/VRRP-MIB.txt
 VRRPV3_URL          := https://raw.githubusercontent.com/acassen/keepalived/v2.2.8/doc/VRRPv3-MIB.txt
-KEMP_LM_URL         := https://kemptechnologies.com/files/packages/current/LM_mibs.zip
 MIKROTIK_URL        := 'https://box.mikrotik.com/f/a41daf63d0c14347a088/?dl=1'
 NEC_URL             := https://jpn.nec.com/univerge/ix/Manual/MIB
-NET_SNMP_URL        := https://raw.githubusercontent.com/net-snmp/net-snmp/v5.9/mibs
-PALOALTO_URL        := https://docs.paloaltonetworks.com/content/dam/techdocs/en_US/zip/snmp-mib/pan-10-0-snmp-mib-modules.zip
-PRINTER_URL         := https://ftp.pwg.org/pub/pwg/pmp/mibs/rfc3805b.mib
-SERVERTECH_URL      := 'https://cdn10.servertech.com/assets/documents/documents/817/original/Sentry3.mib'
-SERVERTECH4_URL     := 'https://cdn10.servertech.com/assets/documents/documents/815/original/Sentry4.mib'
-SYNOLOGY_URL        := 'https://global.download.synology.com/download/Document/Software/DeveloperGuide/Firmware/DSM/All/enu/Synology_MIB_File.zip'
+NET_SNMP_URL        := https://raw.githubusercontent.com/net-snmp/net-snmp/refs/heads/master/mibs/
```

Comment:
Can a tagged release be used?  This could result in builds of `snmp.yml` being non-reproducible.